### PR TITLE
[deprecated]: remove unused/deprecated code

### PR DESCRIPTION
**Pull request by GLEF1X**

### CODE REVIEW

File: openai/api_requestor.py [hunk 1]
```diff
@@ -392,7 +393,7 @@ def handle_error_response(self, rbody, rcode, resp, rheaders, stream_error=False
             )
 
     def request_headers(
```
Comment (hallacy) [inline openai/api_requestor.py:395]: Nit: can you check if the call in line 490 needs to get updated based on this?

**Review by hallacy — APPROVED** [review]
Wow! Thank you. Thats a ton of little details.  How did you find all of these?

File: openai/api_requestor.py (hunk 1 above)
Comment (GLEF1X) [reply openai/api_requestor.py:395]: Yep, thank you for the review. It's indeed a bug that I overlooked. I've just fixed it in [dc8feeb](https://github.com/openai/openai-python/pull/345/commits/dc8feeb933f97e72a0a266a187d9c28ada45d8de)

**Comment by GLEF1X** [comment]
> Wow! Thank you. That's a ton of little details. How did you find all of these?

I was using a couple of linters, and It's a good idea to integrate some into this project. For instance, I saw a pretty good configuration in [evals](https://github.com/openai/evals), so contributors can follow the same code style. Tools like `pre-commit`, `mypy`, `isort` or `ruff` can really help a lot.

**Comment by rattrayalex** [comment]
It's too bad this was never merged; the SDK has been entirely rewritten since.

**Pull request closed without merging**